pjsip/include/pjsip/sip_config.h

```diff
@@ -676,6 +676,24 @@ PJ_INLINE(pjsip_cfg_t*) pjsip_cfg(void)
 #endif
 
 
+/**
+ * Initial timeout interval to be applied to incoming TCP/TLS transports
+ * (i.e. server side) when no valid data received after a successful
```

Comment:
Minor: doc clarification
The word "data" and "message" seem to be used interchangeable throughout the document, which may cause confusion.

According to the RFC 3261, "SIP messages are either requests or responses", while data may not always refer to an entire message. So better clarify and be precise here.

I would also suggest to use word processor or AI to catch grammatical or spelling errors.